stix_shifter/stix_translation/src/modules/bigfix/bigfix_query_constructor.py

```diff
@@ -8,48 +8,33 @@ class RelevanceQueryStringPatternTranslator:
     comparator_lookup = {
         ComparisonExpressionOperators.And: "AND",
         ComparisonExpressionOperators.Or: "OR",
-        ComparisonComparators.GreaterThan: ">",
-        ComparisonComparators.GreaterThanOrEqual: ">=",
-        ComparisonComparators.LessThan: "<",
-        ComparisonComparators.LessThanOrEqual: "<=",
+        # ComparisonComparators.GreaterThan: ">",
```

Comment:
Unless you think we will support these later, it's safe to delete the commented code. It can always be added from the dummy module later.

Reply:
sure.